playground/src/pages/basics/InstancedRigidBody.vue

```diff
@@ -52,28 +52,9 @@ watch(torusInstancedMesh, (mesh) => {
 
     <Suspense>
       <Physics debug>
-        <InstanceRigidBody ref="instanceRBRef" collider="hull">
+        <InstancedRigidBody collider="hull">
           <TresInstancedMesh ref="torusInstancedMesh" :args="[torusKnots, torusKnotsMaterial, 3]" />
```

Comment:
I don't know how R3f handle it, and is not our priority, but If a mess around with 300 instanceMesh my performance drops dramatically, dunno if a small number like 300 should do this or is by the logic, let's keep an eye on this.

Are you ok with the idea of re-think how we handle instancedRigidBody? and create an issue?

Reply:
Yes, it may be caused by the usage of debug mode with lines or something like that.

I'm going to do some tests as well and re-think it if needed

Reply:
Ok anyhow I'll create a GitHub issue to keep track of this one